### PR TITLE
Add tests for docker and kubelet cgroup driver

### PR DESCRIPTION
**Pull request by przemyslavic**
Task #2703.
Useful for testing Kubernetes upgrade #2696.

### CODE REVIEW

**Review by to-bar — DISMISSED** [review]

**Comment by to-bar** [comment]
@przemyslavic You can consider adding this task to changelog :slightly_smiling_face:

**Comment by przemyslavic** [comment]
Done ☺️

File: tests/spec/spec/kubernetes_master/kubernetes_master_spec.rb [hunk 1]
```diff
@@ -186,7 +185,39 @@
     describe command('kubectl describe nodes | grep -i taints') do
       its(:stdout) { should match /<none>/ }
       its(:exit_status) { should eq 0 }
-    end    
+    end
   end
 
 end
+
+describe 'Check the container runtime cgroup driver' do
```
Comment (atsikham) [inline tests/spec/spec/kubernetes_master/kubernetes_master_spec.rb]: In our case container runtime is Docker, and here different thing is checked, so I would change description.

File: tests/spec/spec/kubernetes_node/kubernetes_node_spec.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe 'Check the container runtime cgroup driver' do
```
Comment (atsikham) [inline tests/spec/spec/kubernetes_node/kubernetes_node_spec.rb]: Same here.

**Review by atsikham — CHANGES_REQUESTED** [review]

**Comment by atsikham** [comment]
In our [documentation](https://github.com/epiphany-platform/epiphany/blob/develop/docs/home/CERTIFICATES.md) we have such statement:

```
kubelet.conf is not renewed because kubelet is configured for automatic certificate renewal. To verify that, navigate to /var/lib/kubelet/ and check config.yaml file, where rotateCertificates setting is true by default.
```

We could check `rotateCertificates` here the same way as `cgroupDriver` (in ConfigMap and `/var/lib/kubelet/config.yaml`).

WDYT about adding it here without an additional task?

**Review by atsikham — APPROVED** [review]
Great to see this done so quick!

**Review by erzetpe — APPROVED** [review]

**Pull request merged**